packages/adapter-zhipu/src/requester.ts

```
@@ -136,15 +136,17 @@
-                        text: '',
-                        generationInfo: {
-                            tokenUsage: {
-                                promptTokens: data.usage.prompt_tokens,
-                                completionTokens: data.usage.completion_tokens,
-                                totalTokens: data.usage.total_tokens
+                        message: new AIMessageChunk({
+                            content: '',
+                            response_metadata: {
+                                tokenUsage: {
+                                    promptTokens: data.usage.prompt_tokens,
+                                    completionTokens: data.usage.completion_tokens,
```

ESLint

[error] Insert `⏎·······································`

(prettier/prettier)